tests/platform_tests/link_flap/test_cont_link_flap.py

```diff
@@ -128,6 +139,18 @@ def test_cont_link_flap(self, request, duthosts, nbrhosts, enum_rand_one_per_hws
         memory_output = duthost.shell("show system-memory")["stdout"]
         logging.info("Memory Status at end: %s", memory_output)
 
+        # Record BGP memory status at end
+        end_time_bgp_memory = self.get_bgp_memory_usage(duthost)
+        logging.info("BGP Memory Usage at end: \n%s", end_time_bgp_memory)
+
+        # Calculate diff in BGP memory
+        incr_bgp_memory = int(end_time_bgp_memory) - int(start_time_bgp_memory)
+        logging.info("BGP absolute difference: %d", incr_bgp_memory)
+
+        # Check BGP memory is not increased
+        pytest_assert(incr_bgp_memory <= 0,
```

Comment:
Is this check condition too strict? should we add a threshold to allow for a reasonable range of the increase?

Reply:
Done. Added a threshold same as the generic memory check.